front/app/containers/Admin/projects/project/nativeSurvey/FormResults/messages.ts

```diff
@@ -108,6 +108,15 @@ export default defineMessages({
     id: 'app.containers.Admin.projects.project.survey.analysis.refresh',
     defaultMessage: '{ count } new responses',
   },
+  regenerate: {
+    id: 'app.containers.Admin.projects.project.survey.analysis.regenerate',
+    defaultMessage: 'Regenerate',
+  },
+  backgroundTaskFailed: {
+    id: 'app.containers.Admin.projects.project.survey.analysis.backgroundTaskFailed',
+    defaultMessage:
+      'There was an error generating the AI summary. Please regenerate the summary below.',
```

Comment:
Would it make sense to remove this little "summary" duplication?
```suggestion
      'There was an error generating the AI summary below. Please regenerate it.',
```

Reply:
Are there any other errors that can happen but cannot be fixed by regenerating? In this case, maybe it would make sense to specify it in the message
```suggestion
      'There was an error generating the AI summary below. Please try to regenerate it. If the issue persists, contact us.',
```

Reply:
Not that I have seen. Some of these errors happen because of timeouts and potentially third party api issues. Contact us feels more general and not specific to who the admin should contact. It also indirectly indicates that it might be buggy. Makes sense to remove the duplication to `There was an error generating the AI summary. Please try to regenerate it below.`

Reply:
Makes sense! Good to merge then.

P.S. I copied the "contact us" part from this message https://github.com/CitizenLabDotCo/citizenlab/blob/fac4e7a221f9f737d5acef88878840612ab919f4/front/app/translations/en.json#L175